userspace/libscap/scap_userlist.c

```diff
@@ -168,6 +168,13 @@ int32_t scap_create_userlist(scap_t* handle)
 
 	return SCAP_SUCCESS;
 }
+#else
+#include "windows_hal.h"
```

Comment:
This probably needs to be enclosed in a windows specific ifdef. This part is also used for OS X

Reply:
fixed